.gitignore

```diff
@@ -78,3 +78,4 @@ Tests/images/README.md
 Tests/images/msp
 Tests/images/picins
 Tests/images/sunraster
+/.mypy_cache/
```

Comment:
This section of .gitignore is for "Extra test images installed from pillow-depends/test_images". Better place is "Unit test / coverage reports"